<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttachmentsWidgetClass</class>
 <widget class="QWidget" name="AttachmentsWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AttachmentsWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListView" name="listView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::TopToBottom</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>false</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAddFile">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Images/Add.png</normaloff>:/Images/Add.png</iconset>
   </property>
   <property name="text">
    <string>Add file</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Images/Minus.png</normaloff>:/Images/Minus.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionAddLink">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Images/AddLink.png</normaloff>:/Images/AddLink.png</iconset>
   </property>
   <property name="text">
    <string>Add link</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Images/Rename.png</normaloff>:/Images/Rename.png</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/Images/Export.png</normaloff>:/Images/Export.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
